blaze/compute/numpy.py

```diff
@@ -122,12 +150,12 @@ def compute_up(t, x, **kwargs):
 
 @dispatch(Not, np.ndarray)
 def compute_up(t, x, **kwargs):
-    return ~x
+    return np.logical_not(x)
```

Comment:
This wasn't showing up because you can't construct an expression that would allow you to call `~` on that expression unless it had a `bool` measure